controller/internal/routes/identity_router.go

```diff
@@ -120,7 +120,7 @@ func (r *IdentityRouter) Register(ae *env.AppEnv) {
 		return ae.IsAllowed(r.getPostureData, params.HTTPRequest, params.ID, "", permissions.IsAdmin())
 	})
 
-	ae.Api.IdentityGetIdentityPostureDataFailedServiceRequestsHandler = identity.GetIdentityPostureDataFailedServiceRequestsHandlerFunc(func(params identity.GetIdentityPostureDataFailedServiceRequestsParams, _ interface{}) middleware.Responder {
+	ae.Api.IdentityGetIdentityFailedServiceRequestsHandler = identity.GetIdentityFailedServiceRequestsHandlerFunc(func(params identity.GetIdentityFailedServiceRequestsParams, _ interface{}) middleware.Responder {
```

Comment:
All other changes triggered by changes in the swagger.yml file other than variable renames.